### PR TITLE
defaultbrowser 1.1 (new formula)

### PR DESCRIPTION
**Pull request by ylluminarious**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This is a continuation of https://github.com/Homebrew/homebrew-core/pull/23621 since the developer tagged a new release as requested.

### CODE REVIEW

File: Formula/defaultbrowser.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Defaultbrowser < Formula
+  desc "Command-line tool for getting & setting the default browser"
+  homepage "https://github.com/kerma/defaultbrowser"
+  url "https://github.com/kerma/defaultbrowser/archive/1.1.tar.gz"
+  sha256 "56249f05da912bbe828153d775dc4f497f5a8b453210c2788d6a439418ac2ea3"
+
+  def install
+    system "make"
+    mkdir_p bin.to_s
```
Comment (DomT4) [inline Formula/defaultbrowser.rb]: This should no longer be necessary.

File: Formula/defaultbrowser.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    system "make", "install", "PREFIX=#{prefix}"
+    bin.install "defaultbrowser"
```
Comment (DomT4) [inline Formula/defaultbrowser.rb]: Same comment here.

File: Formula/defaultbrowser.rb (hunk 1 above)
Comment (ylluminarious) [reply Formula/defaultbrowser.rb]: Good catch.

File: Formula/defaultbrowser.rb (hunk 2 above)
Comment (ylluminarious) [reply Formula/defaultbrowser.rb]: Good catch also.

**Comment by ylluminarious** [comment]
@DomT4 Fixed as per your requests.

File: Formula/defaultbrowser.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+class Defaultbrowser < Formula
+  desc "Command-line tool for getting & setting the default browser"
+  homepage "https://github.com/kerma/defaultbrowser"
+  url "https://github.com/kerma/defaultbrowser/archive/1.1.tar.gz"
+  sha256 "56249f05da912bbe828153d775dc4f497f5a8b453210c2788d6a439418ac2ea3"
+
+  def install
+    system "make"
```
Comment (DomT4) [inline Formula/defaultbrowser.rb]: This should be redundant now too. I just neglected to update upstream's `README` when I filed the prior PR.

Comment (ylluminarious) [reply Formula/defaultbrowser.rb]: @DomT4 Oh, ok. Updated accordingly.

**Comment by ylluminarious** [comment]
@DomT4 Anything holding this back now?

**Review by DomT4 — APPROVED** [review]
It's good with me, at least. Punting to ILZ.

**Comment by ylluminarious** [comment]
@ilovezfs Are we good here?

**Comment by fxcoudert** [comment]
Great, thanks @ylluminarious for the pull request

**Pull request closed without merging**